doc/topics/development/logging.rst

```diff
@@ -2,4 +2,21 @@
 Logging Internals
 =================
 
-TODO
\ No newline at end of file
+You can call the logger from custom modules to write messages to the minion
+logs. The following code snippet demonstrates getting access to the logger:
+
+.. code-block:: python
+    import logging
+
+    log = logging.getLogger(__name__)
+
+    log.info('here is some information')
+    log.warning('you should not do that')
+    log.error('it is busted')
```

Comment:
@jacobhammons, since this is based on a comment I wrote without careful thought, I feel somewhat responsible (embarrassed).  Can we change this so that the log statements are Sentence case (first letter is capitalized)?  I could be wrong, but this seems to be the convention in salt logging.  We may even want to explicitly recommend it somewhere.  Also, I was going to make these changes here but later refrained because it seems the document was originally intended to be a programmatic reference on the salt logging system rather than instructions on how to use salt's logging system.  The latter is more important and useful than the former, but we should probably change the title from `Logging Internals` to something like `Logging with Salt`.